### DISCUSSION TITLE
How can I get the feature type from a MVTLayer feature to style it?

### DISCUSSION THREAD

**Discussion by smcallis** [visgl/deck.gl#8736] [Q&A]
I want to change color based on whether a feature is a point, polyline or polygon, but looking at the object that's passed to getLineColor, it doesn't have any of the underlying feature properties set on it, just the layer name:

![image](https://github.com/visgl/deck.gl/assets/804394/2c41f45d-375d-4e80-b3a3-ec1d2a1a12b5)

So how can I access those properties?

**Comment by Pessimistress** [comment]
Your tile source does not contain any additional information.

**Comment by smcallis** [comment]
It has to have the feature type like polyline/polygon in it
somewhere doesn't it?

**Reply by smcallis** [reply]
I ended up solving this by adding a tag with the feature dimension on it.
